cddl/measurement-values-map.cddl

```diff
@@ -15,5 +15,6 @@ measurement-values-map = non-empty<{
   ? &(name: 11) => text
   ? &(cryptokeys: 13) => [ + $crypto-key-type-choice ]
   ? &(integrity-registers: 14) => integrity-registers
+  ? &(measured-component: 15) => measured-component
```

Comment:
15 conflicts with the linear privlevel measurement PR.

Reply:
Oops, apologies.  Will fix.